x/pairing/keeper/pairing_test.go

```diff
@@ -776,6 +777,136 @@ func TestSelectedProvidersPairing(t *testing.T) {
 	}
 }
 
+// Test that the pairing process picks identical providers uniformly
+func TestPairingUniformDistribution(t *testing.T) {
```

Comment:
this test is identical to the next one - except for the extra staking. 
you can refactor the loop part out and both tests would use it, instead of duplication.

Reply:
fixed 0bb8a541